packages/codeeditor/src/editor.ts

```diff
@@ -451,10 +451,18 @@ namespace CodeEditor {
 
     /**
      * Repaint the editor.
+     * 
+     * #### Notes
+     * A repainted editor should fit to its host node.
      */
     refresh(): void;
 
     /**
+     * Resize the editor to fit its host node.
+     */
+    resizeToFit(): void;
```

Comment:
We can call refresh on the codemirror insrance directly on show/attach.  Otherwise it looks goid to me.